File: 1451.重新排列句子中的单词.go
/*
 * @lc app=leetcode.cn id=1451 lang=golang
 *
 * [1451] 重新排列句子中的单词
 */

// @lc code=start
func arrangeWords(text string) string {
	words := strings.Split(text, " ")
	words[0] = strings.ToLower(words[0])
	sort.SliceStable(words, func(i, j int) bool {
		return len(words[i]) < len(words[j])
	})
	words[0] = strings.Title(words[0])
	return strings.Join(words, " ")

}

// @lc code=end

File: 1451.重新排列句子中的单词.py
#
# @lc app=leetcode.cn id=1451 lang=python3
#
# [1451] 重新排列句子中的单词
#

# @lc code=start
class Solution:
    def arrangeWords(self, text: str) -> str:
        text = text.lower()
        text = text.split(' ')
        text.sort(key=lambda x: len(x))
        text[0] = text[0].capitalize()
        return ' '.join(text)
# @lc code=end

